templates/ja_company/js/owl-carousel/owl.carousel.css

```diff
@@ -46,10 +46,10 @@
 .owl-carousel .owl-item .client-item{
 	padding: 0 5px;
 }
-
 .owl-controls .owl-page,
 .owl-controls .owl-buttons div{
 	cursor: pointer;
+	float: left;
```

Comment:
Why do we need it? SVG icons changed something? If so, I suggest we go and find the root cause instead of rewriting on this behavior.

Reply:
from https://github.com/Keyamoon/svgxuse

"._.. IE (9, 10, 11) fails to load symbol-defs.svg in the example above. svgxuse goes through every <use> element. If the element is referencing an external SVG and the browser has failed to load it, the script sends a GET request to grab and prepend the SVG to <body>..._"

by nature polyfills are api's that instert a functionality that a browser natively does not support, so it should work fine. 

remember that font-awesome is also a solution that [supports a range of browsers](https://fontawesome.com/how-to-use/on-the-web/other-topics/browser-support) , we are simply reducing the risk of incompatibility by using a more modern approach with a fallback option.

Reply:
on main page at bottom, under OUR SERVICES there are two swipe buttons (left/right), as I replaced the <i> tag with <svg> here, the alignment of those buttons were glitched. this float fixes that problem.

remember this change is not final, as i am doing a list of heavy optimization and removals. I am putting an effort so that we get a viable and modern design at the end.  the root cause you mention may not worth investigating as it costs time us all, where the layout may/will change and that root cause most probably disappear.